src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -51,6 +62,18 @@ $doc-hero-overlay-background: transparent !default;
 $doc-hero-icon-opacity: 1 !default;
 $doc-hero-icon-color: dark-color(fill-secondary) !default;
 
+.documentation-hero-disabled {
```

Comment:
It looks like these are mostly duplicated from the `documentation-hero` class with minor adjustments. Instead of toggling between the `documentation-hero` and `documentation-hero-disabled` classes, it might be easier to have the `documentation-hero` class always there and only add a  modifier class for when the special dark background is enabled so you don't have to duplicate as much styling—you only need to override a few small things with the modifier class.